src/Tools/TestDiscoveryWorker/Program.cs

```diff
@@ -65,7 +65,7 @@
         await Console.Out.WriteLineAsync($"Discovering tests in {testDescriptor}...").ConfigureAwait(false);
 
         using var xunit = new XunitFrontController(AppDomainSupport.IfAvailable, assemblyFileName, shadowCopy: false);
-        var configuration = ConfigReader.Load(assemblyFileName);
+        var configuration = ConfigReader.Load(assemblyFileName, configFileName: null);
```

Comment:
For reference: A new overload was added that also has optional parameter, causing ambiguity.